libconnman-qt/networkservice.cpp

```diff
@@ -724,6 +719,7 @@ inline QString NetworkService::Private::stringValue(const QString &key)
     return m_propertiesCache.value(key).toString();
 }
 
+// deprecated
```

Comment:
Also here, maybe deprecate in away it is visible in some way?

Reply:
done.